mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -279,6 +285,24 @@ impl From<NumConversError> for UtxoRpcError {
     fn from(e: NumConversError) -> Self { UtxoRpcError::Internal(e.to_string()) }
 }
 
+impl UtxoRpcError {
+    pub fn is_tx_not_found_error(&self) -> bool {
+        if let UtxoRpcError::ResponseParseError(ref json_err) = self {
+            if let JsonRpcErrorType::Response(_, json) = &json_err.error {
+                if json["error"]["code"] == -5 {
+                    return true;
+                };
+                // electrum compatible
+                if json["message"].as_str().unwrap_or_default().contains(NO_TX_ERROR_CODE) {
+                    return true;
+                };
```

Comment:
These lines can be simplified to `return json["error"]["code"] == -5 || json["message"].as_str().unwrap_or_default().contains(NO_TX_ERROR_CODE);`